app/res/values/strings.xml

```diff
@@ -306,6 +306,7 @@
     <string name="keep_changes" cc:translatable="true">SAVE INCOMPLETE</string>
     <string name="go_to_location" cc:translatable="true">Go to Location</string>
     <string name="font_size" cc:translatable="true">Text Font Size</string>
+    <string name="button_font_size" cc:translatable="true">Button Font Size</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/d8071be6a79b86c79dbbe293fdb65271db475f21/app/res/values/strings.xml#L309"></span>The resource <code>R.string.button_font_size</code> appears to be unused</td>
    </tr>
  </tbody>
</table>